content/1_General/Contributing.mdx

```diff
@@ -27,7 +27,7 @@ If you have any questions about how to contribute, feel free to just open an
 [issue](https://github.com/cpinitiative/usaco-guide/issues) or a
 [pull request](https://github.com/cpinitiative/usaco-guide/pulls) and a team
 member will be able to assist you. If you want, you can also
-[join our Discord server](https://discord.gg/UKbyewj2Ft).
+[join our Discord server](https://discord.gg/53zkXQH8).
 
 ![Discord](https://img.shields.io/discord/717477028072521797?label=Discord&style=for-the-badge)
```

Comment:
this should also be updated, but when I try https://img.shields.io/discord/754884077542441002?label=Discord&style=for-the-badge it says `Widget disabled`, not sure how to fix?

Reply:
maybe just remove it then?

Reply:
[__https://img.shields.io/discord/754884077542441002?label=Discord&style=for-the-badge](https://img.shields.io/discord/754884077542441002?label=Discord&style=for-the-badge)

Reply:
https://img.shields.io/discord/754884077542441002?label=Discord&style=for-the-badge

(I just enabled it)

Reply:
alr i added it @thecodingwizard